packages/react-search-ui/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.9.0",
   "description": "A React library for building search experiences",
   "license": "Apache-2.0",
-  "main": "lib",
+  "main": "lib/cjs",
```

Comment:
It seems like maybe this is something @joemcelroy missed in his react-search-ui PR?

In which case I think this should probably be:
```
  "main": "lib/cjs",
  "module": "lib/esm",
  "types": "./lib/esm/index.d.ts",
```

Reply:
Decided to revert this change completely to avoid conflicts with Joe's PR.